src/lib/pages/frontsite/Literature/LiteratureSearch/SearchGuideLink/SearchGuideLink.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Container } from 'semantic-ui-react';
+
+export default class SearchGuideLink extends Component {
+  render() {
+    return (
+      <Container textAlign="left">
+        <Link to="/guide/search">Search guide</Link>
```

Comment:
the url should not be hardcoded, there is a config variable dedicated to the static pages from which you can take the URL